runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -1075,8 +1075,19 @@ J9::CodeGenerator::lowerTreeIfNeeded(
        (node->getSymbol()->castToMethodSymbol()->getRecognizedMethod() == TR::java_lang_invoke_MethodHandle_invokeBasic))
       {
       TR::SymbolReference *vmThreadTempSlotSymRef = self()->comp()->getSymRefTab()->findOrCreateVMThreadTempSlotFieldSymbolRef();
```

Comment:
It would be instructive to have a brief comment here describing what this lowering is doing rather than having to grok the code.  Presumably, this is a meta-argument for invokeBasic?